docs/Documentation/Drivers/Image/Params.md

```diff
@@ -1,2 +1,142 @@
+# VIC Model Parameters
+
 !!! Warning
     Docs not written yet.
+
+# Converting from VIC 4 to VIC 5
+
+We are developing a few scripts to aid users in moving from VIC 4 to VIC 5.  These scripts are currently distributed via [`tonic`](https://github.com/UW-Hydro/tonic).
+
+### Installing Tonic Using Anaconda
+
+The easiest way to install Tonic and its dependencies is to use the [Anaconda Python distribution](https://store.continuum.io/cshop/anaconda/).
+
+To install Anaconda, follow these two simple steps (check to make sure the installer version is the most current)
```

Comment:
You might want to give some guidance as to which python release (2.7 or 3.5), or if it doesn't matter, or if the user has to figure out which version based on other factors, mention that.  The user will see the choice and wonder, if they're not familiar with python.